Makefile

```diff
@@ -10,7 +10,8 @@ pretty:
 	black --line-length 120 .
```

Comment:
Consider running pretty in CI

https://github.com/rAIdiance/chest/blob/main/.github/workflows/ci.yml#L38

Reply:
Black sorts imports slightly differently than isort, which would lead to problems in the CI. Do we use isort in rAIdiance?